### DISCUSSION TITLE
Dedicated Slack Channel for Shopware Frontends

### DISCUSSION THREAD

**Discussion by meeshoogendoorn** [shopware/frontends#66] [Ideas]
Is there gonna be a dedicated slack channel available for Shopware Frontends. Their is some activity on this github discussions section but would it not be great to have a dedicated slack channel in the Shopware Slack Community?

The Shopware PWA channel offered some great help to some community members, also for me.

**Comment by BrocksiNet** [comment]
Feel free to join this [channel](https://shopwarecommunity.slack.com/archives/C050L6NCMGQ) in Shopware Community Slack.

**Reply by meeshoogendoorn** [reply]
Thanks for the quick reply and creation of the channel 😄
